script/src/error.rs

```diff
@@ -0,0 +1,28 @@
+use ckb_error::{Error, ErrorKind};
+use failure::Fail;
+
+#[derive(Fail, Debug, PartialEq, Eq, Clone)]
+pub enum ScriptError {
+    /// The field code_hash in script is invalid
+    #[fail(display = "InvalidCodeHash")]
+    InvalidCodeHash,
+
+    /// The script consumes too much cycles
+    // NOTE: the original name is ExceededMaximumCycles,
+    #[fail(display = "TooMuchCycles")]
+    TooMuchCycles,
```

Comment:
`TooMuchCycles` is really a bad name,  How much is too much?
the original name is much better, why replace?